src/main/java/com/michelin/ns4kafka/controllers/acl/AccessControlListNonNamespacedController.java

```diff
@@ -0,0 +1,34 @@
+package com.michelin.ns4kafka.controllers.acl;
+
+import com.michelin.ns4kafka.controllers.generic.NonNamespacedResourceController;
+import com.michelin.ns4kafka.models.AccessControlEntry;
+import com.michelin.ns4kafka.security.ResourceBasedSecurityRule;
+import com.michelin.ns4kafka.services.AccessControlEntryService;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+import javax.annotation.security.RolesAllowed;
+import javax.inject.Inject;
+import java.util.List;
+
+@Tag(name = "ACLs resource")
+@Controller("/api/acls")
+@RolesAllowed(ResourceBasedSecurityRule.IS_ADMIN)
+public class AccessControlListNonNamespacedController extends NonNamespacedResourceController {
+
+    /**
+     * The ACL service
+     */
+    @Inject
+    AccessControlEntryService accessControlEntryService;
+
+    /**
+     * Get all the ACLs of all namespaces
+     * @return A list of ACLs
+     */
+    @Get
+    public List<AccessControlEntry> listAll() {
```

Comment:
Add unit tests for this new controller (see: [ResourceQuotaNonNamespacedControllerTest.java](https://github.com/michelin/ns4kafka/blob/master/src/test/java/com/michelin/ns4kafka/controllers/ResourceQuotaNonNamespacedControllerTest.java) for example)